Fantasy-Quest/Assets/Scripts/Cutscene/Skip/Hint.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using Configs;
+using DG.Tweening;
+using TMPro;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cutscene.Skip
+{
+    [AddComponentMenu("Scripts/Cutscene/Skip/Cutscene.Skip.Hint")]
+    internal class Hint : MonoBehaviour
+    {
+        private GameplayInput playerInput;
+
+        private TMP_Text textMeshPro;
+
+        [SerializeField]
+        private float fadeHintDuration;
+
+        private void Awake()
+        {
+            playerInput = new GameplayInput();
+            textMeshPro = GetComponent<TMP_Text>();
```

Comment:
Just use `[SerializeField]` with `[Require]` in field, instead of `GetComponent`.
Also if you use `GetComponent` be sure that you add `RequireComponent` to MonoBehaviour.